lib/js/dom.nim

```diff
@@ -1570,7 +1579,9 @@ proc routeEvent*(d: Document, event: Event)
 proc write*(d: Document, text: cstring)
 proc writeln*(d: Document, text: cstring)
 proc querySelector*(d: Document, selectors: cstring): Element
+proc querySelector*(d: Document, selectors: seq[cstring]): Element
 proc querySelectorAll*(d: Document, selectors: cstring): seq[Element]
+proc querySelectorAll*(d: Document, selectors: seq[cstring]): seq[Element]
```

Comment:
where is this documented that a seq would work? https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll mentions string